libs/chain-signatures/contract/src/tee/tee_state.rs

```diff
@@ -93,16 +143,18 @@ impl TeeState {
     ) -> u64 {
         self.votes.vote(code_hash.clone(), participant)
     }
-
+    // Retrieves the current allowed hashes, cleaning up any expired entries.
```

Comment:
Triple backslash for docs.
```suggestion
    /// Retrieves the current allowed hashes, cleaning up any expired entries.
```